avocado/utils/genio.py

```diff
@@ -201,3 +202,24 @@ def write_file_or_fail(filename, data):
     except OSError as details:
         raise GenIOError("The write to %s failed: %s" % (
                          filename, details))
+
+
+def find_pattern(filename,  pattern):
```

Comment:
This `find_pattern` name may give the idea that the function returns the pattern found. I think we have two better options here:
1) rename the function to something like `pattern_matches_in(filename, pattern)` (just a suggestion to ilustrate);
2) In fact return the pattern found. This is the default behaviour of `re.search()` function. Since `search()` returns `None` when there's no match, the way to use the return object can be the same as the current return. So we could only have:
```python
def find_pattern(filename, pattern):
    ...
    with open(filename, 'r') as content_file:
        return re.search(pattern, content_file.read(), re.MULTILINE)
```
What do you think?

Reply:
sure `pattern_matches_in` make more sense will adopt this , i make it return true/false as in avocado-misc test we used this return value (true/false) let me think to adopt this